api/ncf_api_flask_app/views.py

```diff
@@ -12,7 +12,7 @@
 sys.path[0:0] = [NCF_TOOLS_DIR]
 
 default_path = ""
-use_rudder_auth = True
+use_rudder_auth = False
```

Comment:
Put it back to true please :)

Be careful when you select files to commit ( not always use git add -u .